Description: Disable doc generation for coro.nim
Author: Federico Ceratto <federico@debian.org>
Bug: https://github.com/nim-lang/Nim/issues/3823

--- a/web/website.ini
+++ b/web/website.ini
@@ -64,7 +64,7 @@
 srcdoc2: "pure/md5;pure/rationals"
 srcdoc2: "posix/posix"
 srcdoc2: "pure/fenv;pure/securehash"
-srcdoc2: "pure/basic2d;pure/basic3d;pure/mersenne;pure/coro"
+srcdoc2: "pure/basic2d;pure/basic3d;pure/mersenne"
 
 ; Note: everything under 'webdoc' doesn't get listed in the index, so wrappers
 ; should live here
